### PR TITLE
Nx/jest ci bench/test 1

### PR DESCRIPTION
**Pull request by Hotell**
## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/Avatar-mount_base.html">627</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/Avatar-mount.html">675</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/Button-mount_base.html">319</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/Button-mount.html">308</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/Field-mount_base.html">1151</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/Field-mount.html">1139</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/FluentProvider-mount_base.html">711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/FluentProvider-mount.html">704</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/FluentProviderWithTheme-mount_base.html">86</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/FluentProviderWithTheme-mount.html">90</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">41</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">37</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">74</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">80</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/MakeStyles-mount_base.html">869</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/MakeStyles-mount.html">854</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/Persona-mount_base.html">1793</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/Persona-mount.html">1779</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/SpinButton-mount_base.html">1405</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/SpinButton-mount.html">1410</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/SwatchPicker-mount_base.html">1637</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32073/perf-test-react-components/SwatchPicker-mount.html">1715</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

File: .devops/templates/build-test-lint.yml [hunk 1]
```diff
@@ -18,8 +18,8 @@ steps:
       # @fluentui/api-docs is used within apps/public-docsite-resources/just.config.ts,
```
Comment (fabricteam) [inline .devops/templates/build-test-lint.yml:18]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32073&commitId=37c36c254084206ac50267c07d8c6f0689c635ce&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|2| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|5| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default - Dark Mode.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by layershifter** [comment]
Closing for housekeeping 🏡  Feel free to re-open

**Pull request closed without merging**